nucleus/parent/pom.xml

```diff
@@ -829,6 +829,26 @@
                                 <proc>only</proc>
                             </configuration>
                         </execution>
+                        <execution>
```

Comment:
Wouldn't it work with full in default-testcoimpile?
In standard build we have it because it follows some preprocessing if I remember well. Maybe we should try it to see - something should fail ... maybe it can be set even more simple way.

Reply:
The default changed in JDK 23, so that when you specify nothing the annotations are not processed.

Therefore the option to do "only"; so the second execution does the annotations. Optionally we can do "full", but then we can only compile on the very latest JDK versions.

Reply:
You are right here, I did not read the link carefully.